docs/concepts.md

```diff
@@ -196,22 +196,34 @@ and an Oak Application.
 
 ## Remote Attestation
 
-Remote attestation is a core part of Project Oak. When a client connects to an
-Oak Node, the two first establish a fresh ephemeral session key, and then they
-provide assertions to each other over a channel encrypted with such key; the
-client relies on this assertion to determine whether it is connecting to a valid
-version of the Oak Runtime (see below for what constitutes a valid version). In
-particular, the attestation includes a _measurement_ (i.e. a hash) of the Oak
-Module running in the remote enclave, cryptographically bound to the session
-itself.
-
-The client may then infer additional properties about the Oak Module running on
-the remote enclave, e.g. by means of "static attestation" certificates that are
-produced as a byproduct of compiling the Oak Module source code itself on an
-enclave and having the enclave sign a statement that binds the (hash of the)
-compiled Oak Module to some high-level properties of the source code.
-
-TODO: Expand on this.
+Remote attestation is a core part of Project Oak.
+
+When a client connects to an Oak Application (currently via a gRPC Server
+pseudo-Node), the two first establish a fresh ephemeral session key, and then
+they provide assertions to each other over a channel encrypted with such key;
+the client relies on this assertion to determine whether it is connecting to a
+valid version of the Oak Runtime (see below for what constitutes a valid
+version). In particular, the attestation includes a _measurement_ (i.e. a hash)
+of the exact version of the Oak Runtime itself in the remote enclave,
+cryptographically bound to the gRPC session and signed by the hardware root of
+trust for the remote attestation (e.g. Intel in the case of SGX).
+
+This is currently provided by the
+[Enclave Key Exchange Protocol (EKEP)](https://asylo.dev/docs/concepts/ekep.html)
+as part of the [Asylo](https://github.com/google/asylo) framework.
+
+Once the client is satisfied with the remote attestation provided by the remote
+Oak Runtime instance, it may then send data over this channel, in such a way
+that the data is end-to-end encrypted between the client and that particular
```

Comment:
Is this the same E2E encryption as in the paragraph 2 above?

Reply:
yes, reworded.